.github/workflows/install_snippet.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+# Set by GH actions, see
+# https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
+TAG=${GITHUB_REF_NAME}
+
+
+cat << EOF
+## Intsall
+
+See install instructions in [README.md](https://github.com/aspect-build/aspect-cli/blob/${TAG}/README.md).
+
+If you already have [bazelisk](https://github.com/bazelbuild/bazelisk) installed, you can have [bazelisk](https://github.com/bazelbuild/bazelisk) install the Aspect CLI just like it can install the standard Bazel CLI.
+
+Add this to your \`.bazeliskrc\` in your project folder to install Aspect for all developers:
+
+\`\`\`
+BAZELISK_BASE_URL=https://github.com/aspect-build/aspect-cli/releases/download
+USE_BAZEL_VERSION=aspect/${TAG}
+\`\`\`
+EOF
```

Comment:
"When using bazeliskrc to install the Aspect CLI for your project, the `.bazelversion` file or the `BAZEL_VERSION` environment variable will be used to determine which underlying version of Bazel to use."